public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.ts

```diff
@@ -5,7 +5,7 @@ import { AGGREGATION_OPERATORS, RANGE_VEC_FUNCTIONS } from '../../../syntax';
 
 import { CompletionDataProvider } from './CompletionDataProvider';
 import { NeverCaseError } from './NeverCaseError';
-import type { Situation, Label } from './situation';
+import { Situation, Label, getSituation } from './situation';
```

Comment:
Here `Situation` is not used.

Reply:
That change was not meant for this branch.